prompting/tasks/sentiment.py

```diff
@@ -0,0 +1,37 @@
+from prompting.tasks import Task
+
+QUERY_PROMPT_TEMPLATE = """\
+You are a review-generating expert, focusing on making highly reaslistic revies. Your response contains only the review, nothing more, nothing less. You will adhere to a word limit of 250 words. Ask a specific question about the following context:
+{context}
+"""
+
+
+class SentimentAnalysisTask(Task):
+    important_field = ["{context"]
```

Comment:
Is that correct? Not sure if we are missing a `}` or if we have an extra `{`

Reply:
Shouldn't be there at all actually, good catch